src/app/background/index.ts

```diff
@@ -30,12 +30,18 @@ if (NODE_ENV !== 'production') {
 console.info(`BACKEND_ORIGIN "${BACKEND_ORIGIN}"`);
 
 onInstalled.then(installedDetails =>
+  // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
```

Comment:
You probably had to do this because you didn't have the right dependencies for this branch.
You should run `yarn install`  and these errors would probably go away.

Reply:
I did, remove yarn.lock and node_modules then yarn install  but still the same issues...

Reply:
You shouldn't remove your `yarn.lock`!

Reply:
to renew dependecies it's important no ?